### ISSUE TITLE
Aromachemical screen update

### ISSUE THREAD

**Issue by MFernflower** [OpenSourceMycetoma/Adjuvant-Screen#1]
I got a reply back from the company

@mattodd seems like they won't be able to provide the compounds Pro Bono but they are interested nonetheless!

@wwjvdsande or @bebi78 could try and see what of the list they can order and ship using research funding?!

https://shop.perfumersapprentice.com/ is there main website

**Comment by bebi78** [comment]
@MFernflower @mattodd @wwjvdsande I checked the prices for the compounds of the list (lowest available amounts of Perfumers Apprentice, without shipping costs, see below). I think we don't need all of them and ´´high-priced´´ hexadecanolide, for example, can be omitted in my opinion. But at least, bisabolene and ambrettolide should be considered for ordering because of their biological activities.
Galaxolide (4 ml): 3 USD
Calone 1951 (8 g): 11 USD
Javanol (4 ml): 11.25 USD
Ambrettolide (4 ml): 7.75 USD
Maritima (4 ml): 4.75 USD
Ambroxane crystals (8 g): 15 USD
Habanolide (4 ml): 3 USD
Globanone (4 ml): 3 USD
Fructalate (4 ml): 3.50 USD
Scentenal (4 ml): 3 USD
Bisabolene (4 ml): 5.25 USD
Indolarome (8 g): 8 USD
Hexadecanolide (4 ml): 15.25 USD
Total: 93.75 USD

**Comment by MFernflower** [comment]
We could probably omit Hexadecanolide, Maritima, Galaxolide and Fructalate to be honest!
@OpenSourceMycetoma/corecontrib 

Updated list then:

Calone 1951 crystals (8 g): 11 USD
Javanol (4 ml): 11.25 USD
Ambrettolide (4 ml): 7.75 USD
Ambroxane crystals (8 g): 15 USD
Habanolide (4 ml): 3 USD
Globanone (4 ml): 3 USD
Scentenal (4 ml): 3 USD
Bisabolene (4 ml): 5.25 USD
Indolarome crystals (8 g): 8 USD

**Comment by bebi78** [comment]
@MFernflower @mattodd @wwjvdsande OK. I will order these compounds from Perfumers Apprentice, and maritima because I like the combi of terpene with pyridine.

**Comment by MFernflower** [comment]
@bebi78 If you have a chance could you upload a graphic or spreadsheet of all the molecules you purchased?

**Comment by bebi78** [comment]
@MFernflower Here is the word file with the structures of the ordered compounds.

[PerfumersApprentice.docx](https://github.com/OpenSourceMycetoma/Adjuvant-Screen/files/8217101/PerfumersApprentice.docx)

**Comment by MFernflower** [comment]
@dmitrij176 to add these compounds to master list when time permits

**Comment by mattodd** [comment]
Actually I think we need to spread the load a little here. @dmitrij176 has been adding a fair few compounds to the Master List recently. Could one of you @MFernflower or @bebi78 perhaps take a turn? The fields should be self-explanatory, and [the MyOS numbering system](https://github.com/OpenSourceMycetoma/MycetOS-Tech-Ops/wiki/The-Molecule-Numbering-Convention) is hopefully clear?

**Comment by MFernflower** [comment]
@mattodd I don't know if I even have write access to the master list but my
apologies for assigning the work as I did!

**Comment by mattodd** [comment]
Yes, you should be able to edit the [Master List](https://docs.google.com/spreadsheets/d/1YhK-3i2KwuVabo1GbZSgVjAUbavEICCMKi5v-EhNq80/edit?usp=sharing).

**Comment by MFernflower** [comment]
@mattodd @bebi78 I entered the data - fingers crossed I did not make any mistakes

**Comment by bebi78** [comment]
@MFernflower @mattodd @wwjvdsande I'm sorry, there is a problem with the order. Because Perfumer's Apprentice only accepts orders with advance payment, I can't order it because our university does not order anything with advance payment. Any suggestions?

**Comment by MFernflower** [comment]
@bebi78  there are other suppliers for aroma compounds perhaps they could
step in? Could always try to contact perfumer's apprentice via email?

**Comment by bebi78** [comment]
@MFernflower Yes, I wrote an email to Perfumer's Apprentice, but seems to be no other way than advance payment.

**Comment by MFernflower** [comment]
Calone 1951, Habanolide and Scentenal might be available from the manufacture themselves as they have a sample program

They also make muscone and some other terpenoid products too 

https://www.firmenich.com/company/contact-us

@bebi78

**Comment by bebi78** [comment]
@MFernflower Good idea. Maybe these companies like Firmenich, IFF and Givaudan, which produce these fragrances, can provide Wendy with free samples, because only 2-3 mg or 1 mL are needed.

**Comment by MFernflower** [comment]
@bebi78 I don't have an institutional email so probably best for somebody who does to reach out to the chemical companies who make these molecules on an industrial scale 

Also with regard to quantities: Perfume chemists don't normally use the small amounts that drug chemists use so the quantities would be more like 100 mg of the solid products and 500 to 1500 μL of liquid products

**Comment by MFernflower** [comment]
@bebi78 @mattodd I'm going to try to reach out to Firmenich - Wish me luck!

**Comment by bebi78** [comment]
@MFernflower @mattodd Thanks and good luck! I will contact Givaudan next week by mail or phone for free samples of ambrettolide, javanol and bisabolene. It took some time but I found their list of fragrances and ingredients (see below) with numerous interesting compounds such as safranal or jasmone.

[giv-fb-fragrance-ingredients-sustainability-profile-catalogue.pdf](https://github.com/OpenSourceMycetoma/Adjuvant-Screen/files/8237621/giv-fb-fragrance-ingredients-sustainability-profile-catalogue.pdf)

**Comment by MFernflower** [comment]
@bebi78 if you could make up a table of things you find interesting
from Givaudan
that'd help me alot - I think via phone might be better option I'm still
waiting on Firmenich to awnser email but I digress

**Comment by wwjvdsande** [comment]
@MFernflower @bebi78 sorry for not answering before. I was away for a week. Our university also does not pay in advance. But the options you discussed by looking at other companies might work! Will be a lot of fragrances in our assay I assume. I need to take that also in consideration when I perform the test. With some of the plant extracts with fragrances I tested before we sometimes also had volatile compounds in them which could influence the MICs of other compounds.

**Comment by bebi78** [comment]
@wwjvdsande Thanks. I don't think that every compound will be active. If there are two-three hits in this series of fragrances with activity at 20 µM and 100 µM it would be excellent.   
@MFernflower Yes, let me know your opinion about the following Givaudan compounds (terpenoid-type compounds and aromatic compounds related to cinnamaldehyde):
Ambrettolide
Bisabolene
Caryophyllenol
Cinnamyl cinnamate
Dupical
Ebanol
Ethyl safranate
Florhydral
Florocyclene
Folenox
Givescone
Glycolierral
Isolongifolanone
cis-Jasmone
Javanol
Okoumal
Oxyoctaline formate
Safranal
Silvial
Spirambrene
Spirogalbanone
Verdoracine
Vetyvenal

**Comment by MFernflower** [comment]
@bebi78 indeed those look promising although I wasn't expecting Silvial to
look so much like ibuprofen!

I have not gotten a response from Firmenich yet as of writing this message

**Comment by MFernflower** [comment]
@bebi78 @wwjvdsande the compounds I want to get samples of from Firmenich
are solids with a low melting point


I did send the a message a few days ago - maybe if I don't get response
maybe @bebi78 can send one over - example letter is in repository

**Comment by MFernflower** [comment]
@wwjvdsande the compounds I want to get samples of from Firmenich
are solids with a rather low melting point - could just fridge them if they melt during shipping maybe??

I am a few days into sending the message - no response yet from Firmenich - maybe @bebi78 or @mattodd can send one over - example letter to them is in this repository - Maybe I used wrong email or something?!?

**Comment by bebi78** [comment]
@MFernflower I called Givaudan Germany and they will discuss our request for their compounds. They will reply soon. Of course, I will contact them again in case there is a delay. I also called Firmenich Germany today and also wait now for their decision if they can provide us with Calone 1951, Habanolide and Scentenal.

**Comment by MFernflower** [comment]
@bebi78  I had changed over some of the requested compounds from Firmenich since there was some overlap with the ones you were requesting from Givaudan (compare Scentenal and Dupical for example) 

Hopefully this won't be a problem!

Revised Firmenich Request List: 

Calone
Cascalone
Civetone
Cyclopentadecanone
Doremox

**Comment by bebi78** [comment]
@MFernflower OK, I corrected the list of compounds for Firmenich and sent it to them by email. I will call them and Givaudan on Monday again.

**Comment by MFernflower** [comment]
@bebi78 Thanks for keeping up on this - I wonder if my email to Firmenich got stuck in a spam filter somewhere?

**Comment by bebi78** [comment]
@MFernflower I called them today again and both Firmenich and Givaudan told me it will take some days until they can give me a response, but they promised to do so.

**Comment by MFernflower** [comment]
Since we haven't yet gotten any information back from Firmenich  @dmitrij176 maybe could possibly consider making a calone type compound? Maybe one without the methyl group?
![norcalone](https://user-images.githubusercontent.com/3164942/159799575-36421dfe-00db-4e85-bbb5-32dd34412f2c.png)

**Comment by bebi78** [comment]
@MFernflower I haven't received an answer from Firmenich, yet, either. Givaudan forwarded me to IES-Ingredients in France. I hope they are cooperative.

**Comment by MFernflower** [comment]
@bebi78 A company located around 2 hour's from wendy's lab does sell a few interesting compounds including the calone and ambretolide 

https://eng.hekserij.nl/

**Comment by MFernflower** [comment]
IES-Ingredients is the retail and distribution arm of Givaudan from what I have gathered @bebi78

**Comment by bebi78** [comment]
@MFernflower Thanks for the info. De Hekserej seems to be a very promising alternative supplier in NL. Prices are OK. But it also delivers only upon advance payment unless there is a special written agreement. Maybe we can conclude such an agreement. I will check their offered compounds this weekend more carefully if there are further interesting compounds in addition to calone and ambrettolide. I hope their products are OK and no witchcraft/hekserij/Hexerei :-)

**Comment by MFernflower** [comment]
@bebi78 A few interesting compounds from the site to point you in the right direction - Picona (a terpenoid from pine trees maybe?) and Zenolide (an ambrettolide analogue)

**Comment by bebi78** [comment]
@MFernflower OK, I have made the following list of possible interesting compounds by De Hekserij (top ones above the line are those also offered by Firmenich and Givaudan from our previous lists):
Ambrettolide: 12 Euro, 10 ml
Bisabolene: not in pure form, only as component of lime oil or black pepper oil 
Calone: 12.95 Euro, 10 g
Cinnamyl cinnamate: not in pure form, only as component of Davana essential oil 
Ebanol: 5.95 Euro, 10 ml
Ethyl safranate: 9.95 Euro, 5 g
Florhydral: 6.95 Euro, 10 ml
Isolongifolanone/Piconia: 4.95 Euro, 10 ml
Javanol: 6.95 Euro, 1 g
Safranal: 12.95 Euro, 10 ml
Scentenal: 5.95 Euro, 10 ml 
--------------------------------------------------
Bourgeonal: 7.95 Euro, 10 ml
Cedroxyde:  5.95 Euro, 10 ml
Damascone beta: 12.95 Euro, 10 ml (antifungal)
Dihydrojasmone: 7.95 Euro, 10 ml (antifungal)
Floralozone: 3.95 Euro, 10 ml
Hexyl cinnamal: 4.95 Euro, 10 ml
Isoeugenol: 5.95 Euro, 10 ml (antifungal)
Methyl isoeugenol: 5 Euro, 10 ml (antifungal)
Raspberry ketone: 3.95 Euro, 10 g (antifungal)
Sandalore: 6 Euro, 10 ml
Vetiveryl acetate: 12.95 Euro, 5 g
Zenolide: 3.95 Euro, 10 ml

**Comment by bebi78** [comment]
@MFernflower A person from IES contacted me today afternoon. Unfortunately my mail to IES was truncated during the forwarding process and he did not know what compounds we want. So I had to write him our wishlist again this afternoon. I hope that I will get a positive answer from them next week.

**Comment by MFernflower** [comment]
@bebi78 It seems like IES is willing to provide compounds?

I think maybe we give on on Firmenich and order the following from Hekserij:

Zenolide: 3.95 Euro, 10 ml
Calone: 12.95 Euro, 10 g
Cedroxyde: 5.95 Euro, 10 ml
Scentenal: 5.95 Euro, 10 ml
Ambrettolide: 12 Euro, 10 ml

**Comment by MFernflower** [comment]
@bebi78 It seems like IES-Givaudan is willing to provide compounds?

I think maybe we give up on Firmenich and order the following from Hekserij:

Zenolide: 3.95 Euro, 10 ml
Calone: 12.95 Euro, 10 g
Cedroxyde: 5.95 Euro, 10 ml
Scentenal: 5.95 Euro, 10 ml
<strike> Ambrettolide: 12 Euro, 10 ml </strike>  --- I think IES will give us this? 

With regard to needing a written contract - I don't know if the contract needs to be written in Dutch - or if English or German will suffice - Maybe @wwjvdsande could try to reach out/pay and contract the four compounds listed above using institutionally provided money (the cost should be sub 100 euro after taxes and shipping)

**Comment by bebi78** [comment]
@MFernflower We will see whether IES-Givaudan will provide the requested compounds or at least some of them. But at least I'm in contact now with the expert person of IES-Givaudan who deals with such affairs and hope to make an agreement with him.

**Comment by MFernflower** [comment]
@bebi78 Sounds good!!! Hopefully IES-Givaudan will be interested in our project! 

Maybe @wwjvdsande or @MA-Jjingyi  could try to reach out to De Hekserij  by phone or email  https://hekserij.nl/contact/ to see what they can do about the four compounds since they are an under 2 hour drive away from Wendy's laboratory! 

(these are the smallest sizes found on the website - maybe if we only ask for like 2 ml of the liquids and a 1/2 gram of the calone powder maybe they could work something out money wise?) 

Zenolide: 3.95 Euro, 10 ml
Calone: 12.95 Euro, 10 g
Cedroxyde: 5.95 Euro, 10 ml
Scentenal: 5.95 Euro, 10 ml

**Comment by MFernflower** [comment]
@bebi78 has any progress been made

**Comment by bebi78** [comment]
@MFernflower Yes, tonight my contact person from IES-Ingredients answered (see attached files) and he agreed to send the requested compounds for 5 Euros each (for 5 g), in total 115 Euros for the 23 selected compounds. As far as I can see advance payment is not required and payment can be carried out upon delivery of the package. I will establish an account for my university and order the compounds then on Monday. I hope it will work this time.

[IES-Ingredients.docx](https://github.com/OpenSourceMycetoma/Adjuvant-Screen/files/8396974/IES-Ingredients.docx)
[Informations création compte client export-A.pdf](https://github.com/OpenSourceMycetoma/Adjuvant-Screen/files/8396975/Informations.creation.compte.client.export-A.pdf)

**Comment by MFernflower** [comment]
That leaves the 4 or so compounds from the Dutch chemical supplier we mentioned previous @wwjvdsande 


https://hekserij.nl/contact/

**Comment by bebi78** [comment]
@MFernflower @wwjvdsande Yes, these are the four compounds IES does not offer. Personally, I also think hexyl cinnamal might be an interesting alternative or additive for Wendy's published CIN-102 mixture based on more than 80% cinnamaldehyde.

**Comment by MFernflower** [comment]
@bebi78 It's pretty interesting but not unexpected that crude drug research and aromachemical research overlap but then again humans have only recently developed fully synthetic fragrances in the last 150 years or so

**Comment by MFernflower** [comment]
@bebi78 Do you know if it's possible for them to include the molecular weights on the bottles

**Comment by bebi78** [comment]
@MFernflower Thanks. Seems that the order was successful this time. After receiving the package, I will prepare a package for Wendy and will care that the proper information will be placed in the package. I think the description of my samples was not the problem last time anyway since they were quickly tested in the lab.
Because I expect to get 5 g of each sample there will be enough for chemical modifications, and I can send some grams to chemists of the group if there is interest.

**Comment by mattodd** [comment]
Hi - just catching up on this thread, which is now quite long!
Could @MFernflower or @bebi78 consider summarising where we're up to in the wiki for this repo, in just a few lines of rationale and progress?
The compounds that have been ordered - are they the ones that have been added to the Master List? i.e. is the compound list the same as for the first (unsuccessful order)? If not, we should fix.
Out of interest, why is "adjuvant" used in the name of this repository? Aren't these compounds being evaluated for their ability to kill the pathogen, not to enhance an immune response?

**Comment by MFernflower** [comment]
With regards to the adjuvant name this was picked because the paper on the
original paper Wendy and Mickey wrote 10.1371/journal.pntd.0009488 - had
said the original product CIN-102 was a synergist

As to the general rationale - me and @bebi78 saw that since cin-102 was
derived from an herbal remedy that we might as well look around that space
(most synthetic perfume ingredients are  analogues of chemicals isolated
from herbs)

@mattodd

**Comment by MFernflower** [comment]
I think once we get some decent data for in-vivo and in-vitro screening of the plant extracts from Africa, the listed above aroma-chemicals and the natural product derivatives from @bebi78 

We could perhaps package all of that data into a short paper perhaps using the connective tissue of these products being either herbal or derivatives/analogues of herbal products - An example paper title I just came up with out of the top of my head:

"Tapping the perfume tree:  Diverse isolates and analogues of plant natural products show activity against eumycetoma causing fungi" 



Thoughts @OpenSourceMycetoma/corecontrib  ???????

**Comment by wwjvdsande** [comment]
@OpenSourceMycetoma/corecontrib @MFernflower @bebi78 sounds good. It does depend a bit on how many compounds we have to screen this year on all compounds series in the larvae if it is doable of course. At the moment it is still to cold here to start last week we still had snow. We are aiming to start the first week of may this year.

**Comment by MFernflower** [comment]
@bebi78 did the compounds arrive??

**Comment by bebi78** [comment]
@MFernflower Not yet. But IES told me they are working on it.

**Comment by MFernflower** [comment]
@bebi78 I don't think you will need to contact firmenich as @wwjvdsande
plans to order the compounds on in the De_Hekserij.csv file

You could try to reach out to https://www.iff.com/ to hopefully try and tap
the perfume tree once more
￼

**Comment by bebi78** [comment]
@MFernflower @wwjvdsande Thanks, I will check the IFF tree the next days if there is something interesting growing on it. If you have discovered anything there let me know. Because eugenol and cinnamyl alcohol showed interesting preliminary activities (maybe better than cinnamaldehyde) I will try to get some more eugenol and cinnamyl alcohol derivatives from common chemical providers such as Aldrich and TCI. I have found isoeugenol here in our institute, which also has interesting antifungal activities and I can share a sufficient amount of it. Maybe commercially available clove essential oil (7.95 Euros at De Hekserij) might also be worth testing. There can be a synergistic interplay with other ingredients of the essential oil.
A combination of pure eugenol or cinnamyl alcohol with a compound of the fenarimol/ketoxime series might also be promising because I think these compounds attack different fungal targets. 
I will also try to get some semi-synthetic derivatives of eugenol over the next months, e.g., modified compounds of the sort of Ptero-Hepam.

**Comment by MFernflower** [comment]
IFF has this steroid-looking thing called "ambermor ketal crystal" that might be worth a try?  - they also have some alkaloid like compounds (Cinnamalva and Maritima)
@bebi78

**Comment by bebi78** [comment]
@MFernflower Yes, good selection. Interesting structure of ambermor ketal. 
Cinnamalva/cinnamonitrile can be easily obtained from common chemical providers. It would be a good choice for testing, as another close analog of cinnamaldehyde. I don't have cinnamonitrile here, but maybe it is in a chemistry lab storage in London or Sydney? Then I won't need to order it.
Maritima has an interesting pyridine ring, possibly interacting with the iron centers of cytochromes like azoles.

**Comment by bebi78** [comment]
@MFernflower Nootkatone and the three damascones are also interesting IFF fragrances because of their antifungal properties. I will try to call IFF Germany tomorrow and will ask them for ambermor ketal, maritima, nootkatone and the three damascones if you agree.
I have ordered cinnamonitrile (cinnamalva), estragole, and magnolol from TCI and Acros today. 
I'm also thinking of ortho-eugenol and coniferyl alcohol as further interesting and commercially available derivatives.

**Comment by MFernflower** [comment]
@bebi78 I think Wendy's up for screening whatever natural products you can find @wwjvdsande

**Comment by bebi78** [comment]
@MFernflower I have already received cinnamonitrile and ergastrole from Acros. I told IES-Ingredients to send me their samples this month.

**Comment by bebi78** [comment]
Sorry, estragole not ergastrole.

**Comment by bebi78** [comment]
@wwjvdsande @MFernflower What about compounds inhibiting other targets of the mevalonate/cholesterol biosynthesis pathway? Bifonazole (I have it here) inhibits lanosterol demethylase and HMG-CoA reductase. Bisphosphonates such as zoledronic acid (an approved drug for the treatment of bone diseases and bone cancers, 1 g for 32 Euros at TCI) inhibits farnesyl pyrophosphate synthase (I can order it because I plan to use it for a cancer project anyway).

**Comment by wwjvdsande** [comment]
@bebi78 @MFernflower bifonazole and zoledronic acid were not tested yet. We can take them along of course

**Comment by bebi78** [comment]
@MFernflower @wwjvdsande Good news from IES. They informed me today that a first package with ordered samples leaves their warehouse today, the second one with the remaining samples will be dispatched by the end of this week. So I expect to receive the compounds from IES (Givaudan) within the next one-two weeks.

**Comment by MFernflower** [comment]
@wwjvdsande  to order the compounds in De_Hekserij.csv between those compounds and the order @bebi78 is getting from **IES-Givaudan** we should have decent representation of the perfume chemical space - we could still try and follow up with IFF and Symrise but I don't think we need to!

**Comment by bebi78** [comment]
@MFernflower @wwjvdsande IFF Germany forwarded me to Azelis in UK. I'm waiting for a response from the Azelis contact person in Britain now. I requested samples of ambermor ketal crystal, maritima, nootkatone, beta-damascone and delta-damascone.

**Comment by bebi78** [comment]
@MFernflower @wwjvdsande My contact person at Azelis told me they will provide us with the IFF fragrances maritima, nootkatone, beta-damascone and delta-damascone for free. Ambermor ketal is only available as solution, so I removed that from our list.

**Comment by MFernflower** [comment]
@bebi78 what compounds have you received?

**Comment by bebi78** [comment]
@MFernflower @wwjvdsande Good news. Today I received the fragrances from IES/Givaudan, at least most of them (see attached files for structures and SMILES codes). Caryophyllenol, Cinnamyl cinnamate and Vetyvenal were not available. Instead of vetyvenal, they have added Vetynal fine to the package which is a good surrogate I think. 
I have also attached the structures and SMILES of the phenylpropane derivatives I have bought from other providers, or made from eugenol. 
I can dispatch these fragrance molecules on Friday (per UPS again) if it is OK. Then they will arrive in Rotterdam next week.
[IES.Givaudan.docx](https://github.com/OpenSourceMycetoma/Adjuvant-Screen/files/8815116/IES.Givaudan.docx)
[Phenylpropanes.MycetOS.docx](https://github.com/OpenSourceMycetoma/Adjuvant-Screen/files/8815118/Phenylpropanes.MycetOS.docx)

**Comment by MFernflower** [comment]
@bebi78 that's fantastic news - zenolide is another musk you could get from
IFF if you want to ask them for a gram or two

**Comment by bebi78** [comment]
@MFernflower Hm, thought we can get that from de Hekserij.

**Comment by MFernflower** [comment]
@bebi78 I think there is an oxygen missing on the structure of folenox http://www.thegoodscentscompany.com/data/rw1001511.html

**Comment by bebi78** [comment]
@MFernflower Thank you so much. Indeed it is an epoxyde. Corrected word file (corrected structure and SMILES) is attached.
[IES.Givaudan.corrected.docx](https://github.com/OpenSourceMycetoma/Adjuvant-Screen/files/8823241/IES.Givaudan.corrected.docx)

**Comment by bebi78** [comment]
@wwjvdsande @MA-Jjingyi The package was dispatched today, UPS tracking No 1Z99F7F10491300501.

**Comment by MFernflower** [comment]
@wwjvdsande @bebi78 Wendy to order these compounds https://github.com/OpenSourceMycetoma/Adjuvant-Screen/blob/main/De_Hekserij.csv

I took off scentenal as it has a quite reactive aldehyde on it

**Comment by MFernflower** [comment]
@bebi78 @wwjvdsande please note that calone has it's own myos number already: MYOS_00306_00_01

**Comment by MA-Jjingyi** [comment]
> @wwjvdsande @MA-Jjingyi The package was dispatched today, UPS tracking No 1Z99F7F10491300501.

Yep! thanks @bebi78.  we will let you know when we get the package.

**Comment by bebi78** [comment]
@MA-Jjingyi Thanks. UPS informed me it was delivered today. Enjoy the perfumes!

**Comment by bebi78** [comment]
@MFernflower Still haven't received the IFF/Azelis fragrances. They told me they had to wait for one of the four samples they've ordered but received it now. So I hope they will send them out by next week.

**Comment by bebi78** [comment]
@MFernflower Interesting paper about antifungal activity of essential oils. By chance I have thyme EO and thymol in my lab from a different completed project. Oregano and Cassia EO are also interesting. Maybe worth testing.
[fcimb-10-545913.pdf](https://github.com/OpenSourceMycetoma/Adjuvant-Screen/files/8930922/fcimb-10-545913.pdf)

**Comment by bebi78** [comment]
@mattodd @MFernflower @wwjvdsande I contacted an antifungal drug researcher in Nigeria, Prof Emeka Nweze (Dept. of Microbiology, University of Nigeria), who has worked on antifungal drugs against Scedosporium species, among others. Some of the Nigerian plant extracts showed MIC of 2 µg/mL against these fungi according to a conference abstract. Prof Nweze has interest to cooperate with the MycetOS project and wants to learn more about it. Maybe an invitation to join the github group can be sent and infos about the next steps.

**Comment by bebi78** [comment]
@mattodd @MFernflower @wwjvdsande Dr. Ikoni Ogaji from the University of Jos, Nigeria, I met on a pharmacy conference in China some years ago, can also provide antifungal plant extracts for mycetoma testing.

**Comment by bebi78** [comment]
@MFernflower @wwjvdsande I'm not sure if I will receive the four IFF fragrances by next week. But because of the interesting activity of naphthazarin, I plan to send Wendy two more close quinone derivatives and natural dyes next week, plumbagin and 1,4-dihydroxyanthraquinone (quinizarin), to round out the project ´´Tapping the dyer's tree´´. Together with juglone, lawsone, and emodine I think that would give a nice collection and manuscript draft about natural quinones and their strongly varying activities against and selectivites for mycetoma.

**Comment by wwjvdsande** [comment]
@bebi78 @MFernflower, @MA-Jjingyi probably will get a student during the summerbreak to test the fragrances in vitro at least.

**Comment by MFernflower** [comment]
@bebi78 do you have access to buparvaquone? this is a drug I think that is used in India to control protozoan diseases in Cattle

It's another lawsone\henna tree derivative 

![Natural-and-synthetic-bioactive-naphthoquinones-Buparvaquone-4-b-Lapachone-5-and](https://user-images.githubusercontent.com/3164942/175842582-052de3f8-6451-4a0b-bbd6-a65ba7647eb7.png)

**Comment by bebi78** [comment]
@MFernflower Flavio has provided some lapachol-type compounds. I don't know if his compounds were already tested and if they were active @wwjvdsande . But the lawsone derivatives from my lab were rather weak, although they are also quite good antiparasitics. 

https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues/69#issuecomment-1039452768

**Comment by bebi78** [comment]
@MFernflower Buparvaquone seems indeed to be very promising, it is also active in vivo against fungi. I will check if I can get it or prepare it.
[aac.00699-21.pdf](https://github.com/OpenSourceMycetoma/Adjuvant-Screen/files/9009415/aac.00699-21.pdf)

**Comment by bebi78** [comment]
@MFernflower Buparvaquone is available from Chempur, 1 g for 24 Euros. I will order it next week.
@wwjvdsande A new package with additional quinone compounds will be dispatched tomorrow (see attached file). I also added a non-toxic pyridine compound from a DNDi project, which is active against T. cruzi, and which can be considered as an analog of the aminothiazoles.

[Quinones.300622.docx](https://github.com/OpenSourceMycetoma/Adjuvant-Screen/files/9020883/Quinones.300622.docx)

**Comment by MFernflower** [comment]
@bebi78 unrelated to this series but to another series that was put on hold - would it be possible to get some chlorpromazine  for screening use

**Comment by bebi78** [comment]
@MFernflower Yes, I have ca. 100 mg of chlorpromazine hydrochloride here I can share. And I ordered buparvaquone today. 
Azelis informed me that they managed to write all safety data sheets for the four ordered IFF fragrances now. Seems this was the last obstacle before sending it to the EU.

**Comment by MA-Jjingyi** [comment]
Hi, @MFernflower, we have received the compounds. and recently they will be screened by the summer-intern. sorry for replying late.

**Comment by MMos2022** [comment]
Hello @MFernflower, I am Mimi, the summer intern. I was wondering whether the perfume compounds we received from you have MYOS codes already. If not, we would really appreciate it if you could add them to the Excel spreadsheet. Thank you very much.

**Comment by MFernflower** [comment]
Unfortunately I am quite busy with personal medical matters today and
likely won't be able to update the dataset

@bebi78 can you merge the data from the givaudan shipment into the master
spreadsheet if possible

**Comment by MMos2022** [comment]
@MFernflower, @bebi78 Thank you!

**Comment by bebi78** [comment]
@MMos2022 @MFernflower OK, I added MYOS codes and some info to the spreadsheet. Will add more info (MW, formula) when I have more time. Or you add it if you like.

**Comment by MFernflower** [comment]
> @MFernflower Yes, I have ca. 100 mg of chlorpromazine hydrochloride here I can share. And I ordered buparvaquone today. Azelis informed me that they managed to write all safety data sheets for the four ordered IFF fragrances now. Seems this was the last obstacle before sending it to the EU.




Has this shipment been packed - if you could provide a word document like you did for the perfume compounds it would be good

**Comment by bebi78** [comment]
@MFernflower No, not packed yet. I'm waiting for the ordered buparvaquone. Then I will make a list in a word file.

**Comment by bebi78** [comment]
@MFernflower Here is the list of compounds. We used chlorpromazine as drug uptake inhibitor, so I added more such inhibitors like cimetidine, dynasore and endosidin 2 to the package. And I have added thionin and methylene blue as analogs of chlorpromazine. Wendy said she already made tests with methylene blue before but maybe these tests were carried out with a different strain. @wwjvdsande 

[150822.docx](https://github.com/OpenSourceMycetoma/Adjuvant-Screen/files/9338534/150822.docx)

**Comment by bebi78** [comment]
@wwjvdsande You mentioned that you have tested methylene blue as part of photodynamic therapy before. Has it shown antifungal activity? Have you also tested rose bengal as a PDT? Seems to have promising effects on MRSA and Candida. I have plenty of rose bengal B (sodium salt) in case you want to test it. 
I could not find any PDT effects for the naphthoquinones, but some of them showed promising activity against tumors in combination with radiation therapy (XRT). Maybe a way to increase the naphthazarin activity in the larvae?

**Comment by bebi78** [comment]
@wwjvdsande @MA-Jjingyi I have prepared 8 new derivatives for the revision of our ChemMedChem cinnamide manuscript according to the request of one referee (see structures below). Can you test these compounds quickly for the revised version (100 and 25 µM tests)? I can send you the compounds asap.

[Zimt.revision.new.docx](https://github.com/OpenSourceMycetoma/Adjuvant-Screen/files/9706318/Zimt.revision.new.docx)

**Comment by mattodd** [comment]
@bebi78 Great! Happy to add to the Master List and generate the MyOS codes? Helps if we can use this system to avoid all compound ID ambiguity.

**Comment by wwjvdsande** [comment]
@bebi78 we will try to test them as soon as possible. @MA-Jjingyi will be doing some in vitro testing by the end of this week or next week. We can add it if it arrived by then.

**Comment by bebi78** [comment]
@wwjvdsande @MA-Jjingyi Thanks. The package will be dispatched tomorrow by UPS.
@mattodd Excel list was updated.

**Comment by MA-Jjingyi** [comment]
@bebi78 Hi. thanks for your delivery. we got the package. new MIC test will be tested tomorrow. and we will get the final results next Monday 17-10-2022.

**Comment by MFernflower** [comment]
@bebi78 sounds good to me - you need to make certain they haven't added any
stabilizers to there chemicals however

**Comment by MA-Jjingyi** [comment]
@bebi78 Hi, we have tested the susceptibility of strain Mm55 against your compounds. Below you find the results. These results are preliminary as they were only performed once. We need will repeat them as soon as possible. <html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/058620/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/058620/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">



number | name | Mw | weight | 100uM | 25uM
-- | -- | -- | -- | -- | --
1 | Verzimt-pip | 289.33 | 0.0060 | 38.49903 | 81.56863
2 | verzimt-pyr | 275.3 | 0.0050 | 65.78947 | 71.47059
3 | 3br4meozimt-pip | 338.2 | 0.0040 | 3.508772 | 75.68627
4 | 3br4meozimt-pyr | 324.17 | 0.0060 | 63.35283 | 66.56863
5 | i2zimt-pip | 511.1 | 0.0040 | 59.45419 | 69.60784
6 | i2zimt-pyr | 497.07 | 0.0040 | 68.51852 | 70.39216
7 | br2zimt-pip | 417.1 | 0.0060 | 43.56725 | 51.66667
8 | br2zimt-pyr | 403.07 | 0.0050 | 51.85185 | 49.01961



</body>

</html>

**Comment by MA-Jjingyi** [comment]
> @bebi78 Hi, we have tested the susceptibility of strain Mm55 against your compounds. Below you find the results. These results are preliminary as they were only performed once. We need will repeat them as soon as possible.
> 
> <style> </style>
> number	name	Mw	weight	100uM	25uM
> 1	Verzimt-pip	289.33	0.0060	38.49903	81.56863
> 2	verzimt-pyr	275.3	0.0050	65.78947	71.47059
> 3	3br4meozimt-pip	338.2	0.0040	3.508772	75.68627
> 4	3br4meozimt-pyr	324.17	0.0060	63.35283	66.56863
> 5	i2zimt-pip	511.1	0.0040	59.45419	69.60784
> 6	i2zimt-pyr	497.07	0.0040	68.51852	70.39216
> 7	br2zimt-pip	417.1	0.0060	43.56725	51.66667
> 8	br2zimt-pyr	403.07	0.0050	51.85185	49.01961
